### ISSUE TITLE
Which IIAB screencasts / videos are most urgently needed for implementers & teachers?

### ISSUE THREAD

**Issue by holta** [iiab/iiab#1601]
Thanks @tim-moody & @georgejhunt @m-anish if you can help prepare for https://linuxfestnorthwest.org/conferences/2019 in 10 days, when 4-to-6 of us will meet to consider pushing forward George's idea of better vids for implementers & teachers to learn their way around IIAB!

Which [dynamic menuing](https://github.com/iiab/iiab/wiki/IIAB-Menuing) & similar videos are most urgently needed in your opinion?

Thanks for helping us pull together a list here if possible!!

**Comment by holta** [comment]
@georgejhunt & @m-anish can you help others understand the workflow and specific toolchain you each used to make this happen on macOS and Ubuntu-or-similar?

(Eventually leading to your videos within IIAB's YouTube channel here...)
https://www.youtube.com/channel/UC0cBGCxr_WPBPa3IqPVEe3g

**Comment by holta** [comment]
Here are 3 ["Offline YouTubes" (short, friendly screencasts)](https://www.youtube.com/channel/UC0cBGCxr_WPBPa3IqPVEe3g) that nearly every IIAB community around the planet is basically begging for &mdash; so that **backup & duplication of IIAB microSD cards** moves beyond the "Linux priesthood" and is achievable by any serious educator/operator worldwide &mdash; **using her/his existing equipment:**

1) The Desktop version of Raspbian offers an incredibly convenient...Raspberry start icon (top-left of screen) -> Accessories -> SD Card Copier _(Yes, this approach minimizes/truncates onto microSD cards that are just a few bytes smaller!)_
2) Windows laptops can run Beta pre-release "dotNet Disk Imager"  _(But does this minimize/truncate onto microSD cards that are just a few bytes smaller?  In any case Win32 Disk Imager & Etcher.io do not copy all of an IIAB microSD's [ext4] Linux partitions as is needed!)_
3) A more "Rich Tutorial" on how to use `min-sd`, `cp-sd` and `dd` on an old laptop or PC running any recent Linux would deliver genuine grassroots capacity building _(as every NGO claims to offer...but can the NGO's themselves learn to fish...i.e. use these minimization/truncation tools on their macOS MacBooks, if indeed they work there?!)_

IN SHORT: video(s) are urgently needed explaining **"How do I back up, shrink & copy IIAB microSD cards?"** which is currently Number 42 at http://FAQ.IIAB.IO :-)

**Comment by holta** [comment]
Critical, but user-facing (implementer-facing) improvements may have to wait for [IIAB 7.2](https://github.com/iiab/iiab/milestone/7), given that [IIAB 7.1](https://github.com/iiab/iiab/milestone/6) is primarily focused on back-end infra for now.
